pkgs/misc/urbit/ivory-header.nix

```diff
@@ -0,0 +1,25 @@
+{ urbit-src, lib, stdenvNoCC, fetchGitHubLFS, curl, xxd }:
+
+let
+  lfs = fetchGitHubLFS {
+    src = /.
+      + builtins.unsafeDiscardStringContext "${urbit-src}/bin/ivory.pill";
```

Comment:
Why do we need unsafeDiscardStringContext here?

Reply:
Because `fetchGitHubLFS` check for a path and i can not concat path + string if string is in `/nix/store`

Reply:
ok, I see you suggest remove isPath check from `fetchGitHubLFS` function. I'll remove then.